Earthfile

```diff
@@ -47,15 +47,31 @@ update-buildkit:
     SAVE ARTIFACT go.sum AS LOCAL go.sum-fixme  # this is a bug since we can't save to go.sum which was already saved in +deps
 
 
-lint-scripts:
+lint-scripts-base:
     FROM --platform=linux/amd64 alpine:3.13
     RUN apk add --update --no-cache shellcheck
+    WORKDIR /shell_scripts
+
+lint-scripts-misc:
+    FROM +lint-scripts-base
     COPY ./earthly ./scripts/install-all-versions.sh ./buildkitd/entrypoint.sh ./earthly-buildkitd-wrapper.sh \
         ./buildkitd/dockerd-wrapper.sh ./buildkitd/docker-auto-install.sh \
         ./release/envcredhelper.sh ./.buildkite/*.sh \
         ./scripts/tests/*.sh \
-        ./shell_scripts/
-    RUN shellcheck shell_scripts/*
+        ./
+    RUN shellcheck *.sh
```

Comment:
Shellcheck is not in the ARM Alpine repos. The M1 failure did fail on shellcheck. We would need to move that out of the path or force x64 images there